### ISSUE TITLE
sound request: Cat purring

### ISSUE THREAD

**Issue by haggen88** [rafaelmardojai/blanket#256]
Hello, I would like to suggest a cat purring as an additional sound to the current selection.

I attach two audio samples {cuts of the original audio}. I hope @Porrumentzio can analyze them and see if they agree with the parameters used by Blanket.

- 6_Cat, purr.wav   by   '16GPanskaZlochova_Eliska'     [CC0 1.0]
https://freesound.org/people/16GPanskaZlochova_Eliska/sounds/496275/
>  [6_Cat, purr.wav.ogg.zip](https://github.com/rafaelmardojai/blanket/files/8674183/6_Cat.purr.wav.ogg.zip)

- Cat Purring    by    'adda_records'     [CC BY 3.0]
https://freesound.org/people/adda_records/sounds/519977/
> [Cat Purring.mp3.ogg.zip](https://github.com/rafaelmardojai/blanket/files/8674180/Cat.Purring.mp3.ogg.zip)

--

Also I tried to make a sketch of the icon, I hope @jimmac can improve it.
> 
[cat purring.svg.zip](https://github.com/rafaelmardojai/blanket/files/8674202/cat.purring.svg.zip)

**Comment by haggen88** [comment]
@Porrumentzio   I left it at 8 seconds, I guess it's more appropriate now.
[Cat Purring.mp3.ogg.zip](https://github.com/rafaelmardojai/blanket/files/8961356/Cat.Purring.mp3.ogg.zip)

**Comment by daudix-UFO** [comment]
Tested it a little bit, seems good!

**Comment by jimmac** [comment]
While the symbol isn't as geometric as the others, it delivers on cuteness. I like it.

**Comment by TimmB91** [comment]
I totally agree, because the purr of a cat has a very relaxing effect on many people.

**Comment by DancingGeek** [comment]
[cat purring.zip](https://github.com/user-attachments/files/28431211/cat.purring.zip)
This is the 8 second clip from @haggen88 but with the audio adjusted in Ardour to meet the requirements. I believe it works well, as I'm using it myself as a custom sound.

Edit: I've put in a pull request #408
